server/opts.go

```diff
@@ -1680,6 +1680,53 @@ func parseJetStreamForAccount(v interface{}, acc *Account, errors *[]error, warn
 	return nil
 }
 
+func getInterfaceValue(v interface{}) (int64, error) {
```

Comment:
I feel that this function name is too generic and does not give any indication that we are going to get the value for a int64 or a "storage size". Can we have a better name?
Possibly you could "collapse" the two functions into one and be something like `getByteSize()` or `getStorageSize()` (with a line of doc that says what it does)

Reply:
good point, I'll change the name to reflect.